services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -212,6 +213,11 @@ const CMSUserRoutes = ({
                     element={<RateSummary />}
                 />
 
+                <Route
```

Comment:
I believe this route should be in the `SubmissionSideNav` next to `SubmissionSummary`- that's the UI holding the submission details and CMS workflows (currently turned off but it will be turned on soon for Q&A).